src/main/java/com/google/swarm/tokenization/common/FileReaderSplitDoFn.java

```diff
@@ -50,11 +56,11 @@ public void processElement(ProcessContext c, RestrictionTracker<OffsetRange, Lon
         buffer = ByteString.copyFrom(readBuffer);
         readBuffer.clear();
         LOG.debug(
-            "Current Restriction {}, Content Size{}", tracker.currentRestriction(), buffer.size());
-        c.output(Util.readRowSuccess, KV.of(fileName, buffer.toStringUtf8().trim()));
+            "File Read Transform:ReadFile: Current Restriction {}, Content Size{}", tracker.currentRestriction(), buffer.size());
+        c.output(KV.of(fileName, buffer.toStringUtf8().trim()));
       }
     } catch (Exception e) {
-      c.output(Util.readRowFailure, KV.of(fileName, e.getMessage()));
+      LOG.error("File Read Transform:ReadFile: Error processing the file "+ fileName +" - " + Arrays.toString(e.getStackTrace()));
```

Comment:
This is where the multi output should be added. First comment added. So this class should output a Tuple.

Reply:
So should be keep the LOG as a warning here or just take out the log and keep output statement only?

Reply:
I think let's log all in the main class after flatten.